common/views/components/BaseTabs/BaseTabs.tsx

```diff
@@ -52,7 +52,7 @@ type TabPanelProps = {
 const TabPanel = styled.div.attrs((props: TabPanelProps) => ({
   id: props.id,
   role: props.isEnhanced ? 'tabpanel' : undefined,
-  hidden: !props.isActive,
+  hidden: props.isEnhanced ? !props.isActive : undefined,
```

Comment:
I am also a little lost as to what this is doing? Is this because `isActive` is a JS property? Could we not have a prop on the TabPanel called `isHidden` and just pass that through?